src/operator/tensor/elemwise_unary_op.h

```diff
@@ -35,9 +35,10 @@
 #include "../mxnet_op.h"
 #include "../elemwise_op_common.h"
 #include "../../ndarray/ndarray_function.h"
+
 #if MSHADOW_USE_MKL == 1
```

Comment:
I have not set USE_MKL before. Just curious: is blas=MKL also tested in mxnet CI?

Reply:
`MSHADOW_USE_MKL` is widely used in mshadow and mxnet to indicate MKL is used as BLAS library. Yes, `USE_BLAS=mkl` is built and tested in CI:
https://github.com/apache/incubator-mxnet/blob/master/ci/docker/runtime_functions.sh#L375
https://github.com/apache/incubator-mxnet/blob/master/ci/docker/runtime_functions.sh#L553